pages/linux/photorec.md

```diff
@@ -0,0 +1,13 @@
+# PhotoRec
+
+> Deleted file recovery tool.
```

Comment:
This description is a little bit vague. How about 
```suggestion
> Data recovery tool.
```

Reply:
I think data is a bit generic. I think I'd prefer "Deleted file" myself.